config/rbac/operator-role-knative.yaml

```diff
@@ -26,6 +26,7 @@ rules:
   - serving.knative.dev
   resources:
   - services
+  - routes
```

Comment:
Why do we need this?

Reply:
I had cherry-picked this commit from the previous work on upstream main branch, and this just came along. Also, I had noticed missing permission for channels, so I thought this role fixed the issue.